### PR TITLE
Update README.md with new subcommands (analyses) and cmd-line options

### PR DESCRIPTION
**Pull request by danscales**
Also updated with the fact that the subcommand (the analysis) is now always provided as the first argument (and without the --analysis option)

Fixed a help message for the integrated alerts subcommand.

### CODE REVIEW

**Review by dmannarino — APPROVED** [review]
This is great, thanks!

File: README.md [hunk 1]
```diff
@@ -175,8 +208,15 @@ The following options are supported:
...
+|fire_alert_source|string|`firealerts` |URI of fire alert TSV file                                                                                                                   |
+|overwrite        |flag  |all                      |Overwrite output location if already existing                                                                                    |
+|split_features   |flag  |all                      |Split input features along 1x1 degree grid                                                                                       |
+|no_output_path_suffix|flag |all                   |Do not autogenerate output path suffix at runtime                                                                                |
+|intermediate_list_source|flag |`forest_change_diagnostic` analysis |URI of intermediate list results in TSV format                                                                  |
+|contextual_feature_type|flag |`gfwpro_dashboard` analysis |URI of intermediate list results in TSV format                                                                           |
```
Comment (jterry64) [inline README.md:217]: copy and error here, this should be the type of feature for the contextual data

**Review by jterry64 — APPROVED** [review]
This looks good, thanks for updating this!

File: README.md (hunk 1 above)
Comment (danscales) [reply README.md]: Fixed, thanks for catching this!

**Pull request merged**